checks.d/mongo.py

```diff
@@ -326,3 +391,29 @@ def check(self, instance):
             if m in self.RATES:
                 m = self.normalize(m.lower(), 'mongodb') + "ps"
                 self.rate(m, value, tags=tags)
+
+        stat_metrics = filter(lambda x: x.startswith('stats.'), self.METRICS)
+        for s in dbstats:
+            for m in stat_metrics:
+                value = dbstats[s]
+                dbtag = s  # key is db name, so we use that as the tag
+                try:
+                    for c in m.split('.'):
+                        value = value[c]
+                except KeyError:
+                    continue
+
+                # value is now status[x][y][z]
+                assert type(value) in (
+                    types.IntType,
+                    types.LongType,
+                    types.FloatType)
```

Comment:
same as the comment L384, `if not isinstance(...): raise TypeError` would be better.